### PR TITLE
another fix

### PR DESCRIPTION
**Pull request by JanMaartenvanDoorn**
- fixed another issue

### CODE REVIEW

**Review by MartijnCa — CHANGES_REQUESTED** [review]

File: test/unit/postprocessing/test_postprocessing.py [hunk 1]
```diff
@@ -13,21 +13,15 @@
...
+        forecast_sum = pd.DataFrame({"forecast": [-10, -15, -33, 1, 2]})
 
-        forecast_negative_removed["forecast"] = postprocessing.post_process_wind_solar(
-            forecast_positive_sum["forecast"], ForecastType.WIND
-        )
+        forecast_positive_removed = pd.DataFrame()
 
         forecast_positive_removed["forecast"] = postprocessing.post_process_wind_solar(
-            forecast_negative_sum["forecast"], ForecastType.SOLAR
+            forecast_sum["forecast"], ForecastType.SOLAR
```
[the remark this answers is not in the document]
Comment (JanMaartenvanDoorn) [reply test/unit/postprocessing/test_postprocessing.py:22]: No that would not work as most values are set to exactly 0 and that is what we want

File: test/unit/postprocessing/test_postprocessing.py [hunk 2]
```diff
@@ -13,22 +13,16 @@
...
+        forecast_positive_removed = pd.DataFrame()
 
         forecast_positive_removed["forecast"] = postprocessing.post_process_wind_solar(
-            forecast_negative_sum["forecast"], ForecastType.SOLAR
+            forecast["forecast"], ForecastType.SOLAR
         )
 
-        self.assertTrue((forecast_negative_removed["forecast"] >= 0).all())
-        self.assertTrue((forecast_positive_removed["forecast"] <= 0).all())
+        self.assertTrue((forecast_positive_removed["forecast"] <= 0.0).all())
```
Comment (MartijnCa) [inline test/unit/postprocessing/test_postprocessing.py:25]:
```suggestion
        self.assertTrue((forecast_positive_removed["forecast"] <= 0.000001).all())
```

**Review by MartijnCa — APPROVED** [review]
Nice work!

File: test/unit/postprocessing/test_postprocessing.py (hunk 2 above)
Comment (JanMaartenvanDoorn) [reply test/unit/postprocessing/test_postprocessing.py:25]: I keep the 0.0 as this is more strict

**Pull request merged**